packages/core/src/index.ts

```diff
@@ -3,9 +3,21 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+/**
+ * The foundation of a LoopBack app.
+ *
+ * @remarks
+ * Unlike LoopBack 2 and 3, it no longer contains the implementation for
```

Comment:
@raymondfeng , do we want a different remark for this one...seems we'd want  a different one that doesn't involve the sentence `..it no longer contains...` . Just my 2 cents.

Reply:
I've removed that line of remark.